bootstrap/bootstrap.go

```diff
@@ -91,7 +91,11 @@ func RunAndReturnWaitGroup(
 	var wg sync.WaitGroup
 	deferred := func() {}
 
-	lc := logging.FactoryToStdout(serviceKey)
+	// Check is service provided an initial Logging Client to use. If not create one.
+	lc := container.LoggingClientFrom(dic.Get)
+	if lc == nil {
```

Comment:
maybe absorb this logic into `container.LoggingClientFrom(dic.Get)` above?

Reply:
Nope, the get is simple retrieve from DIC. Should not have a side effect.

Reply:
Not sure what you meant. why the reason is something to do with dic.Get? the check is purely on `LoggingClientFrom` function.

Reply:
That is how we retrieve from the DIC. Syntax is a bit strange... ;-)